sdk/appconfiguration/app-configuration/src/internal/cryptoHelpers.browser.ts

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @internal
+ * @ignore
+ */
+export async function sha256Digest(body: any): Promise<string> {
+  const digest = await window.crypto.subtle.digest("SHA-256", new TextEncoder().encode(body || ""));
```

Comment:
should we feature detect `window.crypto.subtle`? For example, in IE it's available, but under `window.msCrypto.subtle` instead of `crypto`. Not sure what browsers we're trying to support with this package.

Reply:
I've done testing (manually) against Edge (new), Chrome and Firefox, which I believe are our core supported browsers.

Reply:
Will do - no reason not to do it right.